src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -945,11 +954,15 @@ private TrackBitrateAllocation(
             int preferredIdx = 0;
             for (RTPEncodingDesc encoding : encodings)
             {
+                if (encoding.getHeight() > this.maxFrameHeight)
+                {
+                    continue;
+                }
                 if (selected)
                 {
                     // For the selected participant we favor resolution over
                     // frame rate.
-                    if (encoding.getHeight() < ONSTAGE_PREFERRED_HEIGHT
+                    if (encoding.getHeight() <= ONSTAGE_PREFERRED_HEIGHT
```

Comment:
This is wrong as it would add 360p at low fps, which is something that we don't want to have.